docs/reference/settings/ml-settings.asciidoc

```diff
@@ -57,8 +58,33 @@ that is greater than this setting value, an error occurs. Existing jobs are not
 affected when you update this setting. For more information about the
 `model_memory_limit` property, see <<ml-apilimits>>.
 
+`xpack.ml.max_open_jobs`::
+The maximum number of jobs that can run on a node. Defaults to `20`.
+The maximum number of jobs is also constrained by memory usage, so fewer
+jobs than specified by this setting will run on a node if the estimated
+memory use of the jobs would be higher than allowed.
+
 `xpack.ml.node_concurrent_job_allocations`::
 The maximum number of jobs that can concurrently be in the `opening` state on
 each node. Typically, jobs spend a small amount of time in this state before
 they move to `open` state. Jobs that must restore large models when they are
 opening spend more time in the `opening` state. Defaults to `2`.
+
+[float]
+[[advanced-ml-settings]]
+==== Advanced machine learning settings
+
+These settings are for advanced use cases; the default values are generally 
+sufficient:
+
+`xpack.ml.max_anomaly_records`:: (<<cluster-update-settings,Dynamic>>) 
+The maximum number of records that are output per bucket. The default value is 
+`500`.
+
+`xpack.ml.max_percent_date_errors`:: (<<cluster-update-settings,Dynamic>>) 
```

Comment:
It turns out that this setting and `xpack.ml.max_percent_out_of_order_errors` are never used anywhere.  (They used to be used in the old Prelert product.)

So they should be removed completely from the docs and code.